bench_scripts/awq_e2e.py

```
@@ -0,0 +1,19 @@
+    llm = LLM(model="QuantTrio/Qwen3.5-9B-AWQ", tensor_parallel_size=2, dtype="float16",
+              enforce_eager=True, gpu_memory_utilization=0.90, max_model_len=4096,
+              max_num_seqs=64, **{"language_model_only": True})
+    # correctness
+    o=llm.generate(["The capital of France is"], SamplingParams(max_tokens=24,temperature=0))
+    print("CORRECT:", repr(o[0].outputs[0].text), flush=True)
+    # decode latency bs=1
+    prompt="Write a long detailed essay about the history of computing."
+    sp=SamplingParams(max_tokens=128,temperature=0,ignore_eos=True)
+    llm.generate([prompt], SamplingParams(max_tokens=8,temperature=0,ignore_eos=True))
+    for B in [1,8,32]:
+        t0=time.time(); oo=llm.generate([prompt]*B, sp); dt=time.time()-t0
```

Ruff (0.15.20)

[error] Multiple statements on one line (semicolon)

(E702)